### ISSUE TITLE
Cannot create Job group from Job create UI flow

### ISSUE THREAD

**Issue by cegue** [iteratec/OpenSpeedMonitor#34]
Hello,

I am running a local OSM version 4.5.1 (commit [c82d5c4](https://github.com/iteratec/OpenSpeedMonitor/blob/master/grails-app/assets/javascripts/job/edit.js#L432)).

While creating a new Job using Chrome, Job Group creation always fails ending with Job Group entry as _undefined_. Developer console prints out:
```
Uncaught TypeError: b.find(...).tagit is not a function
    at createJobGroup (edit.unminified.js:696)
    at HTMLButtonElement.onclick (1:1106)
createJobGroup @ edit.unminified.js:696
onclick @ 1:1106
```
Pointing to [this line in current code baseline](https://github.com/iteratec/OpenSpeedMonitor/blob/master/grails-app/assets/javascripts/job/edit.js#L432).

There is no associated stack trace from the running docker compose stdout shell console.

**To reproduce:**
1. Navigate to *Jobs*
2. Click on *+Create Job*. You will get on console:
```
application.unminified.js:1890 [Vue warn]: Cannot find element: #threshold
application.unminified.js:1890 [Vue warn]: Error in beforeMount hook: "TypeError: Cannot read property 'value' of undefined"
(found in <Root>)
application.unminified.js:1977 TypeError: Cannot read property 'value' of undefined
```
3. Fill in the Job form and click on *+Create New*. You will get:
```
edit.unminified.js:696 Uncaught TypeError: b.find(...).tagit is not a function
    at createJobGroup (edit.unminified.js:696)
    at HTMLButtonElement.onclick (create:1106)
```
**Behavior**
The Job Group is not created and remains as *undefined* for the new Job.

**Expected behavior**
The Job Group is created and available to the new Job.

**Comment by j0weiss** [comment]
Thanks for that detailed bug report. We'll look into that.
A workaround until this is fixed would be to create the job group in the job group controller first and then you can select it while creating a new job.
You find the job group controller under the menu point "All Controller" in the lower part of the sidebar. You have to be logged in as admin to see this menu point.

**Comment by cegue** [comment]
Thanks @j0weiss for sharing the workaround. I edited the title to be more in line with the scope of the issue.

**Comment by j0weiss** [comment]
We just released version [v4.5.2](https://github.com/iteratec/OpenSpeedMonitor/releases/tag/v4.5.2) where we fixed the bug.
Thanks again for reporting!
